main/SQLHelper.h

```diff
@@ -344,140 +346,142 @@ class CSQLStatement
+	bool BackupDatabase(const std::string& OutputFile);
+	bool RestoreDatabase(const std::string& dbase);
 
 	// Returns DeviceRowID
-	uint64_t UpdateValue(int HardwareID, const char *ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel, int nValue,
-			     std::string &devname, const bool bUseOnOffAction, const char *User = nullptr);
-	uint64_t UpdateValue(int HardwareID, const char *ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
-			     const char *sValue, std::string &devname, const bool bUseOnOffAction, const char* User = nullptr);
-	uint64_t UpdateValue(int HardwareID, const char *ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel, int nValue,
-			     const char *sValue, std::string &devname, const bool bUseOnOffAction, const char* User = nullptr);
-	uint64_t UpdateValueLighting2GroupCmd(int HardwareID, const char *ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
-					      int nValue, const char *sValue, std::string &devname, const bool bUseOnOffAction, const char* User = nullptr);
-	uint64_t UpdateValueHomeConfortGroupCmd(int HardwareID, const char *ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
-						int nValue, const char *sValue, std::string &devname, const bool bUseOnOffAction, const char* User = nullptr);
+	uint64_t UpdateValue(int HardwareID, const char* ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel, int nValue,
+		std::string& devname, const bool bUseOnOffAction, const char* User = nullptr);
+	uint64_t UpdateValue(int HardwareID, const char* ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
+		const char* sValue, std::string& devname, const bool bUseOnOffAction, const char* User = nullptr);
+	uint64_t UpdateValue(int HardwareID, const char* ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel, int nValue,
+		const char* sValue, std::string& devname, const bool bUseOnOffAction, const char* User = nullptr);
+	uint64_t UpdateValueLighting2GroupCmd(int HardwareID, const char* ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
+		int nValue, const char* sValue, std::string& devname, const bool bUseOnOffAction, const char* User = nullptr);
+	uint64_t UpdateValueHomeConfortGroupCmd(int HardwareID, const char* ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
```

Comment:
You could use a global substitution
```suggestion
	uint64_t UpdateValueHomeComfortGroupCmd(int HardwareID, const char* ID, unsigned char unit, unsigned char devType, unsigned char subType, unsigned char signallevel, unsigned char batterylevel,
```

Reply:
i have not developped this function ?
do you want that i rename this function?

Reply:
ok rename done